### ISSUE TITLE
The font size of equations in tutorial is too small

### ISSUE THREAD

**Issue by zasdfgbnm** [pytorch/pytorch#15202]
The tutorial at: https://pytorch.org/tutorials/beginner/blitz/autograd_tutorial.html#gradients
![image](https://user-images.githubusercontent.com/1032377/49982017-a95cb500-ff28-11e8-9465-7b0da9ce39c4.png)

**Comment by zasdfgbnm** [comment]
@zou3519
Browser is: Chromium Version 70.0.3538.110 (Official Build) Arch Linux (64-bit)
Operating System: Arch Linux 
KDE Plasma Version: 5.14.4
Qt Version: 5.11.2
KDE Frameworks Version: 5.52.0
Kernel Version: 4.19.4-arch1-1-ARCH
OS Type: 64-bit
Processors: 20 × Intel® Core™ i7-6950X CPU @ 3.00GHz
Memory: 94.3 GiB of RAM

**Comment by zasdfgbnm** [comment]
I don't know what causes the problem, but in the below screenshot, there is a circled CSS saying
```CSS
display: inline-block; position: relative; width: 6.737em; height: 0px; font-size: 52%
```

Manually changing it to `font-size: 100%;` would restore the font size to normal...

![image](https://user-images.githubusercontent.com/1032377/49983855-2725be00-ff33-11e8-9ec7-1de8df1902b8.png)

**Comment by brsoff** [comment]
https://5c1420f1f7907c558e51f188--shiftlab-pytorch-tutorials.netlify.com/beginner/blitz/autograd_tutorial.html#gradients

Are you still seeing the issue on that version? The `minScaleAdjust` of Mathjax's [HTML-CSS output processor](http://docs.mathjax.org/en/latest/options/output-processors/HTML-CSS.html#the-html-css-output-processor) has been bumped from `50` to `100`. Mathjax will try to fit the size of the equation to the surrounding text so I'm unsure why it would be rendering so small for you but this setting might keep it from doing so automatically.

**Comment by zasdfgbnm** [comment]
@brsoff It's much better now:
![image](https://user-images.githubusercontent.com/1032377/50029350-78c85a00-ffc0-11e8-9476-6431abf918a0.png)

**Comment by PradeepSinghMakwana** [comment]
> Are you still seeing the issue on that version? The `minScaleAdjust` of Mathjax's [HTML-CSS output processor](http://docs.mathjax.org/en/latest/options/output-processors/HTML-CSS.html#the-html-css-output-processor) has been bumped from `50` to `100`. 

The link to HTML-CSS part is broken maybe [this](https://docs.mathjax.org/en/latest/output/html.html#html-output) he was pointing to ...
